import-service/catalogBatchProcess.js

```diff
@@ -0,0 +1,38 @@
+const AWS = require("aws-sdk");
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+const sns = new AWS.SNS();
+
+exports.handler = async (event) => {
+  const records = event.Records;
+
+  for (const record of records) {
+    const messageBody = JSON.parse(record.body);
+
+    const product = {
+      id: messageBody.productId,
+      title: messageBody.title,
+      description: messageBody.description,
+      price: messageBody.productPrice,
+    };
+
+    const params = {
+      TableName: "Products",
+      Item: product,
+    };
+
+    try {
+      await dynamodb.put(params).promise();
+      console.log(`Created product: ${product.id}`);
+
+      // Publish an event to the SNS topic
+      const snsParams = {
+        Message: `Product created: ${product.title}`,
+        TopicArn: "arn:aws:sns:eu-west-1:947103499873:ProductTopic",
```

Comment:
Would be cool to add all `arn`s to serverless.yml as environment variables and use those in lambdas through `process.env.YOUR_VARIABLE`